yb-voyager/cmd/analyzeSchema.go

```diff
@@ -964,19 +991,17 @@ func analyzeSchemaInternal() utils.Report {
 	sourceObjList = utils.GetSchemaObjectList(msr.SourceDBConf.DBType)
 	initializeSummaryMap()
 	for _, objType := range sourceObjList {
-		var filePath string
-		if objType == "INDEX" {
-			filePath = filepath.Join(schemaDir, "tables", "INDEXES_table.sql")
+		var sqlInfoArr []sqlInfo
+		filePath := utils.GetObjectFilePath(schemaDir, objType)
+		if objType != "INDEX" {
+			sqlInfoArr = createSqlStrInfoArray(filePath, objType)
 		} else {
-			filePath = filepath.Join(schemaDir, strings.ToLower(objType)+"s")
-			filePath = filepath.Join(filePath, strings.ToLower(objType)+".sql")
+			sqlInfoArr = createSqlStrInfoArray(filePath, objType)
+			otherFPaths := utils.GetObjectFilePath(schemaDir, "PARTITION_INDEX")
+			sqlInfoArr = append(sqlInfoArr, createSqlStrInfoArray(otherFPaths, "PARTITION_INDEX")...)
```

Comment:
Was this a bug that you essentially fixed? index queries are spread across multiple files?

Reply:
yeah right, we were not considering the other index files, although the use cases are less but still possible.